samples/rest/BingAutosuggestV7.py

```diff
@@ -1,45 +0,0 @@
-#Copyright (c) Microsoft Corporation. All rights reserved.
-#Licensed under the MIT License.
-
-# -*- coding: utf-8 -*-
```

Comment:
I see those are missing in new files. I don't know if those lines are needed or not but you can check with Abe.